setup.py

```diff
@@ -10,7 +10,9 @@
 here = path.abspath(path.dirname(__file__))
 
 
-PANDAS = "pandas~=1.3"  # to be able to use XMLDataSet and pandas integration with fsspec
+PANDAS = (
+    "pandas~=1.3"  # to be able to use XMLDataSet and pandas integration with fsspec
+)
```

Comment:
Is this really necessary? `black` hasn't complain on any other branches. If it's a real problem I'd rather move the inline comment above the line.